print_job_queue/migrations/0001_initial.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2022-04-24 11:12
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DraftPrintJob',
+            fields=[
+                ('job_id', models.AutoField(primary_key=True, serialize=False)),
+                ('state', models.IntegerField(choices=[(0, 'PENDING'), (1, 'PROCESSING'), (2, 'DONE')])),
+                ('is_valid', models.BooleanField(default=True)),
+                ('handling_worker', models.CharField(blank=True, max_length=25)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='FinalPrintJob',
+            fields=[
+                ('job_id', models.AutoField(primary_key=True, serialize=False)),
+                ('state', models.IntegerField(choices=[(0, 'PENDING'), (1, 'PROCESSING'), (2, 'DONE')])),
+                ('is_valid', models.BooleanField(default=True)),
+                ('handling_worker', models.CharField(blank=True, max_length=25)),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.CreateModel(
+            name='PrintedDraftDocument',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('file_path', models.FilePathField(match='*.pdf', path='/usr/src/app', recursive=True)),
```

Comment:
Can we use `BASE_DIR` here instead of `/usr/src/app` like what we do in `models.py`?

Reply:
Done.